src/lib/input/input-container.ts

```diff
@@ -101,8 +101,13 @@ export class MdInputDirective {
 
   /** Whether the element is disabled. */
   @Input()
-  get disabled() { return this._disabled; }
-  set disabled(value: any) { this._disabled = coerceBooleanProperty(value); }
+  get disabled() {
```

Comment:
It likely does, but since controls do not keep track of individual validators, it won't have the same fix.